src/nativeAssetManager.js

```diff
@@ -227,12 +271,45 @@ export function newNativeAssetManager(win) {
     }
 
     if (data.message === 'assetResponse') {
-      const body = win.document.body.innerHTML;
-      const newHtml = replace(body, data);
+      const body = win.document.body.innerHTML,flag = (typeof win.pbNativeData !== 'undefined');
```

Comment:
Can you split this `flag` out to a new declaration please.

Reply:
Done